WooCommerce/WooCommerceTests/ViewRelated/ReusableViews/EmptyStateViewController/EmptyStateViewControllerTests.swift

```diff
@@ -0,0 +1,194 @@
+
+import Foundation
+import XCTest
+
+@testable import WooCommerce
+
+/// Test cases for `EmptyStateViewController`.
+///
+final class EmptyStateViewControllerTests: XCTestCase {
+
+    func testItHidesAllConfigurableElementsByDefault() throws {
+        // Given
+        let viewController = EmptyStateViewController()
+
+        // When
+        _ = try XCTUnwrap(viewController.view)
+
+        let mirror = try self.mirror(of: viewController)
```

Comment:
I learned about it yesterday. 😄 I'm a bit wary of it because reflection can be slow. It's definitely useful if we really need to add unit tests to views.